scripts/gen.py

```diff
@@ -216,6 +222,11 @@ class AtenXlaTypeDefault {{
 }}  // namespace torch_xla
 """
 
+_XLAPREAUTOGRAD_FALLTHROUGH_HEADER = """TORCH_LIBRARY_IMPL(_, XLAPreAutograd, m) {
```

Comment:
This has only one use. I think we can avoid making global declarations.
Code is easier to follow (no need to go back and lookup what something is), and cleanups tend to not leave stray code around 😉